sdks/go/README.md

```diff
@@ -17,26 +17,87 @@
     under the License.
 -->
 
-# Go SDK
+# Go SDK Overview
 
 The Apache Beam Go SDK is the Beam Model implemented in the [Go Programming Language](https://go.dev/).
 It is based on the following initial [design](https://s.apache.org/beam-go-sdk-design-rfc).
+Below describes requirements, how to run examples, execute tests, and contribute to the Go SDK.
 
-## How to run the examples
+_A note on Beam specific terminology used in this README._
 
-**Prerequisites**: to use Google Cloud sources and sinks (default for
-most examples), follow the setup
-[here](https://beam.apache.org/documentation/runners/dataflow/). You can
-verify that it works by running the corresponding Java example.
+_This README uses minimally necessary Beam related terminology to help you determine requirements, usage and
+contribution to the Go SDK. A [definitions](#definitions) section below provides short definitions for you to achieve
+these aims._
 
-The examples are normal Go programs and are most easily run directly.
-They are parameterized by Go flags.
-For example, to run wordcount on the Go direct runner do:
+# Requirements
 
+Aside from the obvious [go](https:/go.dev), you will need to clone the
```

Comment:
Nit: Phrasing. There's no need to call out something as obvious. Simply state the obvious requirement.


```suggestion
An up to date version of Go installed. See https://go.dev/doc/install for instruction.

Experience using the Go programming language is strongly recommended. The best place to learn how to program Go is from the Go learning resources. Completing the tutorials https://go.dev/doc/tutorial/getting-started and https://go.dev/doc/tutorial/create-module is a great way to start.
```

Side nit: The URL was missing a /.